infra/resources/modules/commons/containers.tf

```diff
@@ -45,3 +45,65 @@ resource "azurerm_cosmosdb_sql_container" "subscription_history" {
     }
   }
 }
+
+resource "azurerm_cosmosdb_sql_container" "activations" {
+  name                  = "activations"
+  resource_group_name   = azurerm_resource_group.data_rg.name
+  account_name          = module.cosmosdb_account.name
+  database_name         = module.cosmosdb_sql_database_trial.name
+  partition_key_path    = "/trialId"
+  partition_key_version = 2
+
+  autoscale_settings {
+    max_throughput = 1000
+  }
+
+  indexing_policy {
+    included_path {
+      path = "/*"
+    }
+
+    excluded_path {
+      path = "/\"_etag\"/?"
+    }
+
+    composite_index {
+      index {
+        path  = "/trialId"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/type"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/isActive"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/id"
+        order = "Descending"
+      }
+    }
+
+    composite_index {
+      index {
+        path  = "/trialId"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/type"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/id"
+        order = "Ascending"
+      }
```

Comment:
**question**: What use case does this index support? Maybe I'm wrong, but it seems unnecessary to me.

Reply:
I added the `/id` index for error. The index should be (`trialId`, `type`) to help for query where you need to get the `job` for a trial.
What do you think?

Reply:
I had some tests, and the query of the previous comment is supported by the index already defined. I'm going to remove this index.